src/components/report/ReportHeader.svelte

```diff
@@ -7,15 +7,16 @@
   import { reportFilename } from "../../utils/reportFilename.js";
   import { getCatalog } from "../../utils/getCatalogs.js";
 
-  $evaluation.title = $evaluation["product"]["name"] + " Accessibility Conformance Report";
+  $evaluation.title = $evaluation["product"]["name"];
 
   export let download = false;
   let catalog = getCatalog($evaluation.catalog);
 </script>
 
-<Header>{$evaluation.title}</Header>
+<Header><span>{$evaluation.title}</span> Accessibility Conformance Report</Header>
+
+<p>Format: <b>VPAT 2.4Rev WCAG</b></p>
```

Comment:
More generally we could say it is "Based on: "

It isn't strictly following 2.4, but then generally they don't. Everyone modifies the template to do as they wish.

Reply:
Ya #10 is the issue to tackle this one.